recipes/pasta/meta.yaml

```
@@ -37,12 +56,22 @@ test:
     - clustalw2 -help | grep OPTIONS
     - hmmalign -h | grep HMMER
     - hmmbuild -h | grep HMMER
-    - muscle -version | grep MUSCLE
+    - muscle -version | grep -i MUSCLE
     - prank -help | grep prank
-    - raxml -h | grep raxmlHPC
-    - raxmlp -h | grep raxmlHPC
+    - raxml -h | grep -i raxml
+    - raxmlp -h | grep -i raxml
+    - export PASTA_LOGGING_LEVEL=DEBUG; export PASTA_LOGGING_FORMAT=RICH; run_pasta.py -i $PREFIX/share/pasta/data/small.fasta
+    # we need support from upstream to fix this special execution for OSX specific - export PASTA_LOGGING_LEVEL=DEBUG; export PASTA_LOGGING_FORMAT=RICH; run_pasta.py -i $PREFIX/share/pasta/data/small.fasta --tree-estimator raxml
 
 about:
   home: https://github.com/smirarab/pasta
-  license: GNU General Public License v3 or later (GPLv3+)
+  license: GPLv3
```

```suggestion
  license: GPL-3.0-or-later
```
Please use a SPDX identifier, i.e., `GPL-3.0-only` or `GPL-3.0-or-later`.